app/directives.js

```diff
@@ -0,0 +1,74 @@
+// Directives
+airTrafficControlApp.directive('aircraftDetails', function(){
+   return{
+      templateUrl:'directives/aircraftDetails.html',
+      scope:{
+         aircraftObject: "="
+         // this is causing two way binding, not sure

```

Comment:
What are you trying to accomplish; you seem to be hardcoding this in the directive definition, vs maybe it being passed as a parameter to the directive? or do you already have the scope implied? this is very confusing.